src/_style.scss

```diff
@@ -296,8 +296,15 @@ select.form-control {
 .tooltip-shadow {
   box-shadow: 0px 5px 15px 0px rgba(0,0,0,0.3) !important;
 }
+
+.password-tooltip {
+  line-height: 2;
```

Comment:
why are we updating the line height? Just use x-small class and it will adjust automatically to 1.25rem.

<img width="637" alt="Screenshot 2022-07-13 at 4 38 50 PM" src="https://user-images.githubusercontent.com/40633976/178725188-38bfcce4-f68b-413d-875c-bae6d6da0815.png">